tuple-table/src/main/java/com/iluwatar/tuppletable/DBConnection.java

```diff
@@ -0,0 +1,51 @@
+package com.iluwatar.tuppletable;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DBConnection {
+
+    /**
+     * @should @return connection
+     * @throws SQLException
+     * @throws ClassNotFoundException
+     */
+    public Connection getConnection() throws SQLException, ClassNotFoundException, IOException {
+        FileInputStream fis=new FileInputStream("src/main/resources/connection.prop");
+        Properties p=new Properties();
+        p.load (fis);
+        Class.forName((String) p.get ("Dname"));
+        Connection conn = DriverManager.getConnection(
+                (String) p.get ("URL"), (String) p.get ("Uname"), (String) p.get ("password"));
+        System.out.println("Connected database successfully...");
+        p.clear();
+        fis.close();
```

Comment:
Use try-with-resources to close automatically

Reply:
I have removed reading from property file as its in memory implementation now, so doesnt make sense to take an overhead just to save that in property file.